### PR TITLE
attach select Linux packages to release

### PR DESCRIPTION
**Pull request by qrkourier**
oldest and newest distro version for RedHat (RPM) and Ubuntu (DEB), and one intermediate Ubuntu 16.04 LTS to fill in the compatibility gap.

### CODE REVIEW

File: docker/Dockerfile.copy [hunk 1]
```diff
@@ -37,6 +37,7 @@ COPY --from=fetch-ziti-artifacts /etc/ssl/certs/ca-certificates.crt /etc/ssl/cer
 COPY --from=fetch-ziti-artifacts /tmp/${ZITI_TUNNELER_BIN} /usr/local/bin
 COPY ${DOCKER_BUILD_DIR}/docker-entrypoint.sh /
 RUN chmod +x /docker-entrypoint.sh
+RUN mkdir -p /ziti-edge-tunnel
```
Comment (dovholuknf) [inline docker/Dockerfile.copy:40]: did this sneak in? doesn't seem relevant to the pr's title? :)

File: .github/workflows/cpack.yml [hunk 2]
```diff
@@ -123,17 +131,27 @@ jobs:
...
+      - name: upload release artifacts
+        if: ${{ github.event_name == 'release' && startsWith(github.ref, 'refs/tags/v') }}
+        id: get_release
+        uses: softprops/action-gh-release@v1
+        with:
+          draft: false
+          prerelease: false
+          fail_on_unmatched_files: true
+          files: |
+            ./build/ziti-edge-tunnel-${{ matrix.distro.name }}-${{ matrix.distro.version }}-${{ matrix.arch }}.${{ matrix.distro.type }}
```
Comment (dovholuknf) [inline .github/workflows/cpack.yml:157]: at the end of the day - how many things are going to be added to the 'release'?  are there going to be two more entries? one .deb and one .rpm?

[ziti-edge-tunnel-Darwin_x86_64.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Darwin_x86_64.zip)
[ziti-edge-tunnel-Linux_arm.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Linux_arm.zip)
[ziti-edge-tunnel-Linux_arm64.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Linux_arm64.zip)
[ziti-edge-tunnel-Linux_x86_64.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Linux_x86_64.zip)
[ziti-edge-tunnel-Windows_x86_64.zip](https://github.com/openziti/ziti-tunnel-sdk-c/releases/download/v0.18.17/ziti-edge-tunnel-Windows_x86_64.zip)
[Source code (zip)](https://github.com/openziti/ziti-tunnel-sdk-c/archive/refs/tags/v0.18.17.zip)
[Source code (tar.gz)](https://github.com/openziti/ziti-tunnel-sdk-c/archive/refs/tags/v0.18.17.tar.gz)

File: .github/workflows/cpack.yml [hunk 3]
```diff
@@ -18,18 +26,18 @@ jobs:
           - name: ubuntu
             version: "22.04"
             type: deb
-          - name: ubuntu
-            version: "20.04"
-            type: deb
+          # - name: ubuntu
```
[the remark this answers is not in the document]
Comment (qrkourier) [reply .github/workflows/cpack.yml:29]: Hehe, I thought about that when I was re-reading it today. However, these lines are intentionally retained because commenting these lines is how those builds are in/excluded.

File: .github/workflows/cpack.yml (hunk 2 above)
Comment (qrkourier) [reply .github/workflows/cpack.yml:157]: This is what it looks like with the additional Linux package artifacts in the release page: https://github.com/qrkourier/ziti-tunnel-sdk-c/releases/tag/v0.100.5

Comment (dovholuknf) [reply .github/workflows/cpack.yml:157]: i really don't want to add extra "thought" here for the user. let them download 'the one linux one' and call it a day and it should be 'the latest and greatest, get with the program if it doesn't work for you' imo.  when are we going to publish to some sort of repo (and doc the usage, i assume it'll be needed)

Comment (qrkourier) [reply .github/workflows/cpack.yml:157]: The fulcrum of the argument against this unpopular PR seems to be that it introduces ambiguity to the decision point a user faces when they navigate to the GitHub Releases page for tSDK. Today, there's no such ambiguity. You simply choose the correct CPU architecture and OS.

The current plan is to move forward with the unstable distribution repo for YUM first, then APT. I'll close this PR unmerged.

**Comment by qrkourier** [comment]
> The current plan is to move forward with the unstable distribution repo for YUM first, then APT. I'll close this PR unmerged.
